schema.json

```
@@ -85,38 +93,55 @@
             "$ref": "#/definitions/license"
         },
         "license-url": {
-            "$ref": "#/definitions/url"
+            "$ref": "#/definitions/url",
+            "description": "The URL of the license text under which the software or dataset is licensed (only for non-standard licenses not included in the SPDX License List)."
         },
         "preferred-citation": {
-            "$ref": "#/definitions/reference"
+            "$ref": "#/definitions/reference",
+            "description": "When citing the work, cite this reference instead."
+        },
+        "references": {
+            "description": "Reference(s) to other creative works.",
+            "items": {
+                "$ref": "#/definitions/reference"
+            },
+            "minItems": 1,
+            "type": "array",
+            "uniqueItems": true
         },
         "repository": {
-            "$ref": "#/definitions/url"
+            "$ref": "#/definitions/url",
+            "description": "The URL to the software or dataset in a repository (when the repository is neither a source code repository nor a build artifact repository)."
         },
         "repository-code": {
-            "$ref": "#/definitions/url"
+            "$ref": "#/definitions/url",
+            "description": "The URL to the software in a source code repository."
         },
         "repository-artifact": {
-            "$ref": "#/definitions/url"
+            "$ref": "#/definitions/url",
+            "description": "The URL to the software in a build artifact/binary/release repository."
```

```suggestion
            "description": "The URL to the software in a build artifact/binary repository."
```

Would delete this here to avoid ambiguous use (e.g., for things like Zenodo URLs). IMO this is for stuff like PyPI/bintray only.